Flow.Launcher.Core/Flow.Launcher.Core.csproj

```
@@ -57,6 +57,7 @@
     <PackageReference Include="FSharp.Core" Version="7.0.300" />
     <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="2.3.2" />
     <PackageReference Include="squirrel.windows" Version="1.5.2" NoWarn="NU1701" />
+    <PackageReference Include="StreamJsonRpc" Version="2.15.29" />
```

```suggestion
    <PackageReference Include="StreamJsonRpc" Version="2.15.29" />
```

[v2.16.36](https://github.com/microsoft/vs-streamjsonrpc/releases/tag/v2.16.36)

---

yes, let's change it